app/controllers/katello/api/registry/registry_proxies_controller.rb

```diff
@@ -48,7 +49,7 @@ def optional_authorize
 
     def registry_authorize
       @repository = find_readable_repository
-      return true if request.method == 'GET' && @repository && @repository.environment.registry_unauthenticated_pull
+      return true if request.method == 'GET' && @repository && @repository.registry_unauthenticated_pull
```

Comment:
Is this an intentional change? This column doesn't exist, and there's no delegation to the env.